app/services/invoices/refresh_draft_service.rb

```diff
@@ -74,10 +74,10 @@ def call
         # NOTE: In case of a refresh the same day of the termination.
         invoice.fees.update_all(created_at: invoice.created_at) # rubocop:disable Rails/SkipsModelValidations
 
-        return result if tax_error?(calculate_result.error)
-
-        if invoice.should_update_hubspot_invoice?
-          Integrations::Aggregator::Invoices::Hubspot::UpdateJob.perform_later(invoice: invoice.reload)
+        unless tax_error?(calculate_result.error)
+          if invoice.should_update_hubspot_invoice?
+            Integrations::Aggregator::Invoices::Hubspot::UpdateJob.perform_later(invoice: invoice.reload)
+          end
```

Comment:
Was this about the rubocop error but unrelated to this PR?
```
Rails/TransactionExitStatement: Exit statement return is not allowed. Use raise (rollback) or next (commit).```

I'm not sure this is what we expect. I was wondering if we didn't expect the transaction to be aborted 🤔 🤷‍♂️

Reply:
Maybe we should fix it in a separate PR so we can check what's the intended behavior (commit or rollback).